lib/util/cache/repository/impl/base.ts

```diff
@@ -105,8 +106,14 @@ export abstract class RepoCacheBase implements RepoCache {
     }
   }
 
+  private stringify(): string {
+    const sortedData = sortObject(this.data);
+    const stringifiedData = JSON.stringify(sortedData);
+    return stringifiedData;
+  }
+
   async save(): Promise<void> {
-    const jsonStr = JSON.stringify(this.data);
+    const jsonStr = this.stringify();
```

Comment:
i think we should simply use the new fast stable stringify dependency

Reply:
Please point me the exact dependency name, I'll add it

Reply:
https://github.com/renovatebot/renovate/blob/b23b886b6e4c4ac1bcee6ae03a26a60933e28d4f/package.json#L209 🤗

Reply:
🤣🤣🤣